geode-redis/src/main/java/org/apache/geode/redis/internal/executor/key/RedisKeyCommandsFunctionInvoker.java

```diff
@@ -74,18 +73,18 @@ public String type(ByteArrayWrapper key) {
 
 
   @Override
-  @SuppressWarnings("unchecked")
   public boolean rename(ByteArrayWrapper oldKey, ByteArrayWrapper newKey) {
+    while (true) {
```

Comment:
I think the "while (true)" loop should be removed. I think it is a remnant of the tryLock code. As it is now I don't see that it would ever loop.

Reply:
Truth, Darrel. Just removed it!